softgroup/services/profile/profile-impl/src/main/java/com/softgroup/profile/impl/handler/ContactsSyncHandler.java

```diff
@@ -23,6 +29,21 @@ public String getName() {
 
     @Override
     public Response<ContactsSyncResponse> doHandle(Request<ContactsSyncRequest> request) {
-        return null;
+        ContactsSyncRequest requestData = request.getData();
+        ContactsSyncResponse contactsSyncResponse = new ContactsSyncResponse();
+
+        String profileId = request.getRoutingData().getProfileId();
+        List<ContactEntity> contactEntities = requestData.getAddedContactEntities();
+        contactEntities.addAll(requestData.getRemovedContactEntities());
```

Comment:
@VadymPolyanski Return response with NOT_IMPLEMENTED status instead of using this junk code.